acquisitions/src/main/resources/thunderjet/mod-orders/features/unopen-and-change-fund-distribution.feature

```diff
@@ -23,28 +23,29 @@ Feature: Unopen and change fund distribution
     * def orderId = callonce uuid3
     * def poLineId = callonce uuid4
 
-
-  Scenario: Create a fund and budget
+  @Positive
+  Scenario: Unopen and change fund distribution
+    # 1. Create a fund and budget
     * configure headers = headersAdmin
     * call createFund { 'id': '#(fundId)', 'ledgerId': '#(globalLedgerId)'}
     * callonce createBudget { 'id': '#(budgetId)', 'fundId': '#(fundId)', 'allocated': 1000, 'statusExpenseClasses': [{'expenseClassId': '#(globalPrnExpenseClassId)','status': 'Active'}]}
 
 
-  Scenario: Create a composite order
+    # 2. Create a composite order
```

Comment:
lets use print instead of commenting .
This will help in troubleshouting when tests run on separate env and we could not run them locally.

Reply:
in this case we will be able to read full test flow from logs

Reply:
Comments are easier to read in cucumber report and code except for console logs.
![image](https://github.com/folio-org/folio-integration-tests/assets/113523904/d36b1573-cb8f-4d85-af72-60d76c27aeb3)
using #
![image](https://github.com/folio-org/folio-integration-tests/assets/113523904/8c8a2e97-09f3-411f-b051-b175b940f745)

Reply:
I don't know of a practical way to use cucumber reports when running tests locally (they don't appear in files like `overview-features.html`). It's also convenient to use the console log. Would adding an empty comment before the print help to separate the steps ? Or maybe a comment like "# Step" ?

Reply:
1) It is not configured in acq-models, it is possible to implement. In mod-consortia last part there will be link to cucumber report. Example from mod-consortai tests https://github.com/folio-org/folio-integration-tests/pull/1359#issuecomment-2124120046
2) From my prespective title should be only in Scenario. In other places need description about flow what are we doing, what is situation with data. and why this step. With comments, it is easier to do so, with print more force to write just title

Reply:
@azizbekxm I would like us to implement this before we start changing prints into comments, because I would like to see how it changes my workflow. For instance I am currently able to click on a test and see the log right away while the other tests are still being executed - I am not sure I would be able to do that with cucumber reports.

Reply:
Of course, there is changes after implemention cucmber reports. You will no longer see the test title, there will be only one test title OrdersApiTest and link to cumumber report. To Run individual tests you should comment other one. In first attempt it is strange, but need to think. I am urging to move to cucumber report right now, we can think about that of course.

if not productive we can skip to implement cucumber because mod-orders has many tests that may make hard to test individual report. You can test this function running mod-consortia and there will be print.

Comments and Cucumber different suggestion and don't depend to each other. Even we are not moving to cucumber, we should still use comments instead of print. I head one advantage of print is to be able to search scenarios in logs, but error tells which line we have problem.

Reply:
So the consensus is to use print instead of a comment? I will implement the changes now

Reply:
Updated `cancel-order.feature`, and tried to include most recommendations